oaknational/Oak-Web-Application
# [CUR-974] Change examboard references to ks4Option instead

## Description

Music year: 1993

- List of changes
- Where relevant, examboardSlug has been updated to ks4OptionSlug to maintain consistency since pathways have been introduced. 

## How to test

1. Go to https://deploy-preview-2919--oak-web-application.netlify.thenational.academy
2. The curriculum site should retain functionality with no change
3. Curriculum downloads should also be the same

## Checklist
